### ISSUE TITLE
using tslint cli as npm script has inconsistent path behavior for windows and linux

### ISSUE THREAD

**Issue by acdcjunior** [palantir/tslint#1639]
### Bug Report
- **TSLint version**: `~3.15.1`
- **TypeScript version**: `~2.0.3`
- **Running TSLint via**: CLI via NPM script

TypeScript code being linted: (irrelevant)
with `tslint.json` configuration: (irrelevant)
#### `package.json` configuration:

``` json
  "scripts": {
    "unquoted": "tslint src/**/*.ts",
     "quoted": "tslint 'src/**/*.ts'"
  },
```
#### Actual behavior

Unconsistend behavior when running @ windows or linux (and not working at all on windows).
#### Expected behavior

Consistent behavior across platforms.
### Workaround

Use both, I guess:

``` json
  "scripts": {
    "unquoted_and_quoted": "tslint src/**/*.ts && tslint 'src/**/*.ts'"
  },
```
# How to reproduce:

Check readme and files: https://github.com/acdcjunior/tslint-windows-path-inconsistency
(pasting README here for reference:)
## Unquoted path (`npm run unquoted`)
- @windows 10
  - All good: three linting errors!
  
  ``` bash
  $ npm run unquoted
  
  > tslint-windows-path-inconsistency@1.0.0 unquoted D:\github\tslint-windows-path-inconsistency
  > tslint src/**/*.ts
  
  src/FirstLevel.ts[1, 12]: " should be '
  src/second/SecondLevel.ts[1, 12]: " should be '
  src/second/third/ThirdLevel.ts[1, 12]: " should be '
  
  npm ERR! Windows_NT 10.0.14393
  ...
  ```
- @centos 7
  - **Only one error when three are expected!**
  
  ``` bash
  [box@box tslint-windows-path-inconsistency]$ npm run unquoted
  
  > tslint-windows-path-inconsistency@1.0.0 unquoted /box/github/tslint-windows-path-inconsistency
  > tslint src/**/*.ts
  
  src/second/SecondLevel.ts[1, 12]: " should be '
  
  npm ERR! Linux 3.10.0-229.el7.x86_64
  ...
  ```
## Quoted path  (`npm run quoted`)
- @windows 10
  - **Bad: no error whatsoever!**
  
  ``` bash
  $ npm run quoted
  
  > tslint-windows-path-inconsistency@1.0.0 quoted D:\github\tslint-windows-path-inconsistency
  > tslint 'src/**/*.ts'
  
  $
  ```
- @centos 7
  - All good!
  
  ``` bash
  [box@box tslint-windows-path-inconsistency]$ npm run quoted
  
  > tslint-windows-path-inconsistency@1.0.0 quoted /box/github/tslint-windows-path-inconsistency
  > tslint 'src/**/*.ts'
  
  src/FirstLevel.ts[1, 12]: " should be '
  src/second/SecondLevel.ts[1, 12]: " should be '
  src/second/third/ThirdLevel.ts[1, 12]: " should be '
  
  npm ERR! Linux 3.10.0-229.el7.x86_64
  ...
  ```

**Comment by IllusionMH** [comment]
@acdcjunior you can use double quotes `"` (escaped) as workaround to ensure consistency between platforms.
```
"scripts": {
  "lint": "tslint \"src/**/*.ts\""
},
```

I have idea how to fix it, but need more time to ensure that fix for running TSLint from npm scripts won't break anything when it used directly from terminal.

**Comment by JKillian** [comment]
Hi @acdcjunior, [this answer](http://stackoverflow.com/a/40394034/3124288) and [this comment](https://github.com/palantir/tslint/issues/1226#issuecomment-217892560) should provide context for the different behavior you're seeing.

When running in centos unquoted, perhaps you don't have the `globstar` option enabled in your shell and thus the glob isn't matching the files you expect. @IllusionMH's recommendation is good, use double-quotes so that node-glob process the glob.
